tools/topology/topology2/cavs-mixin-mixout-hda.conf

```diff
@@ -102,7 +102,7 @@ Object.Pipeline {
 			}
 			Object.Widget.eqiir.1 {
 				Object.Control.bytes."1" {
-					name '4 Main capture Iir Eq'
+					name '$ANALOG_CAPTURE_PCM Iir Eq'
```

Comment:
Since doing edits, could you also convert Iir to capitals because it's an acronym, IIR.